.dockerignore

```diff
@@ -0,0 +1,74 @@
+# From kuma .gitignore
+!static/fonts/.htaccess
+!build/assets/css/README.md
```

Comment:
My reasoning was you could do a quick check: ``diff .gitignore .dockerignore``.  But that falls apart for the kumascript ``.gitignore``. And maybe we want the ``.htaccess`` file someday. Maybe.

OK, removing them.